xbmc/Autorun.cpp

```diff
@@ -106,10 +106,8 @@ bool CAutorun::PlayDisc(const std::string& path, bool bypassSettings, bool start
   if (mediaPath.empty())
     mediaPath = path;
 
-#ifdef TARGET_WINDOWS
   if (mediaPath.empty() || mediaPath == "iso9660://")
-    mediaPath = g_mediaManager.TranslateDevicePath("");
-#endif
+    mediaPath = g_mediaManager.GetDiscPath();
```

Comment:
@Voyager1 Sorry, IMO my change is okay and keeps Windows code intact. Please take a look at the implementation of CMediaManager::GetDiscPath() - https://github.com/xbmc/xbmc/blob/master/xbmc/storage/MediaManager.cpp#L557. For Windows, it does nothing alse then TranslateDevicePath(""). ;-)